reference/payment_processing.yml

```
@@ -1098,74 +1126,75 @@ components:
           issue_month: 1
           issue_year: 0
           issue_number: 0
+      x-internal: false
       properties:
         type:
-          description: Type to classify this payment instrument
-          example: card
           type: string
           default: card
+          example: card
+          description: Type to classify this payment instrument (required)
         cardholder_name:
-          description: Cardholder's full name
           type: string
           minLength: 1
+          description: Cardholder's full name (required)
         number:
-          description: Credit card number
           type: string
           minLength: 1
+          description: Credit card number (required)
         expiry_month:
-          description: Expiry month of this card
           type: integer
+          format: int32
           minimum: 1
           maximum: 12
-          format: int32
+          description: Expiry month of this card (required)
         expiry_year:
-          description: Expiry year of this card
           type: integer
           format: int32
+          description: Expiry year of this card (required)
         verification_value:
           type: string
-          description: Verification value of this card (CVV or last_4)
           minLength: 3
           maxLength: 4
+          description: Verification value of this card (CVV or last_4)
```

```suggestion
          description: Verification value of this card (CVV)
```
Confirmed by engineering a while back -- this is always the CVV, which is 4 digits for Amex-flavored cards.